docs/_static/config_examples/example-manifest.yml

```diff
@@ -44,3 +44,31 @@ applications:
                         uri: http://api.system.pcf.local
                         port: 80
                         auth_disabled: false
+
+                      # Required to run as a Service Broker
+                      status:
+                        user: user
+                        pass: pass
+
+      # Only required if going to push controller as a Service Broker
+      SERVICE_BROKER_CONFIG: |
+                             plans:
+                              - description: plan for policy A,
+                                name: planA,
+                                virtualServer:
+                                  - policies:
+                                    - policyA
+                                  - profiles:
+                                    - profileA
+                                  - sslProfiles:
+                                    - sslProfileA
+                                pool:
+                                  balance: ratio-member
+                                  healthMonitors:
+                                    - name: monitorA
```

Comment:
I don't think this one needs the "name" key; it should be the path to a health monitor on the BIG-IP, like `/Common/http.get`.

Reply:
(unless Dan led me astray)

Reply:
It does need the name key this was incorrect before. So luckily you requested those changes. You are right though, if you are only using the name key you would provide it with the full path so I'll update this to /Common/http.get

Reply:
Thanks! I'll update the example in the solution docs PR.